pkg/apis/acme/v1/types_issuer.go

```diff
@@ -495,6 +495,12 @@ type ACMEIssuerDNS01ProviderAzureDNS struct {
 
 	// +optional
 	Environment AzureDNSEnvironment `json:"environment,omitempty"`
+
+	// +optional
+	ManagedIdentityClientID string `json:"managedIdentityClientID,omitempty"`
+
+	// +optional
+	ManagedIdentityResourceID string `json:"managedIdentityResourceID,omitempty"`
```

Comment:
Our policy is that new fields should be added to all versions of the API, and you'll find that the tests fail unless you do.
Also please document these new fields and document the undocumented existing fields if you are able.

Reply:
I've added the fields to all api versions and also added comments, let me know if I should change anything

Reply:
Great. If you run `./hack/update-all.sh` it will update the CRD yaml templates with your new fields.